ng/src/web/components/form/select.js

```diff
@@ -91,7 +91,7 @@ class Select extends React.Component {
       className,
       disabled = false,
       items,
-      menuPosition,
+      menuPosition = 'adjust',
```

Comment:
what's the reason for the change? i don't think that makes sense. it breaks e.g. the powerfilter

Reply:
For the MultiSelect it made sense to have a menu position that looks decent by default. I thought it to be a good idea to have that for Select as well. I didn't recognize this breaks the powerfilter.